chips/nrf52832/src/interrupt_service.rs

```diff
@@ -0,0 +1 @@
+pub type Nrf52832InterruptService = nrf52::interrupt_service::Nrf52InterruptService;
```

Comment:
I think it would be clearer to not add this. Since the type is contained in the nrf52832 crate, this could be re-created later if a 832-specific interrupt needed to be handled. This would also make what is happening in nrf52832/chip.rs more clear.

Reply:
Resolving as outdated.